documentation/0045-node-folders/index.md

```diff
@@ -60,7 +60,7 @@ const isFile = fileName => {
 
 fs.readdirSync(folderPath).map(fileName => {
   return path.join(folderPath, fileName)).filter(isFile)
-}
+})
```

Comment:
This isn't right. The filter map closes after the path.join() function but the `}` definitely out of place.